tools/cppcheck/suppressions-list

```diff
@@ -2,3 +2,4 @@ wrongmathcall:tests/unit-libm/test-libm.inc.h
 variableScope:jerry-libm/*.c
 invalidPointerCast:jerry-libm/*.c
 commaSeparatedReturn:*
+ConfigurationNotChecked:jerry-core/ecma/operations/ecma-regexp-object.c
```

Comment:
Is this still needed? (There have been quite some changes and I was hoping that there was no need for this anymore.)

Reply:
Updated the PR without this line and still got the `ConfigurationNotChecked` message on travis. Should I re add this line and update the PR again?

Reply:
It's interesting to see that ConfigurationNotChecked persists but travis is green. The logs show that this message is an information level diagnostic, which apparently does not cause cppcheck to fail eventually. So, as I'd keep the suppression list as short as possible, I'd suggest not to re-add this line.